### ISSUE TITLE
Consider publishing a new version to crates.io

### ISSUE THREAD

**Issue by ebroto** [amzn/askalono#47]
Hi, thanks for this awesome crate!

We started using askalono in [onefetch](https://github.com/o2sh/onefetch) and realized in [this issue](https://github.com/o2sh/onefetch/issues/123) that with the latest changes in askalono we can detect correctly an ISC license, but with the version published in crates.io (0.3.0) it's not working properly.

It seems that we can't publish on crates.io if we rely on unpublished versions, so we were wondering if there would be a new version released anytime soon.

Thanks again!

**Comment by jpeddicord** [comment]
Yeeeeeeeppp, you caught me. I've been on some mental block of "gotta make a 1.0 release" and "it's gotta be perfect" but it's better to release rapidly than not at all.

I believe there's a couple small backwards-incompatible API changes, so I think what I'll do is cut a 0.4.0, with all changes up to now and get that out the door. I've set some time aside for myself to do that tomorrow morning, but might try to do that today too.

Thanks for the prod -- really, I've needed to just do this. :)
